tests/PySys/software-management-end-to-end/sm-fake-install-remove/run.py

```diff
@@ -0,0 +1,60 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate end to end behaviour for the dummy plugin for installation and remove
```

Comment:
How is this test different from the fruits plugin test?

Reply:
Here we use the universal dummy plugin to emulate a fruits plugin. I also added a file that explains the usage and a sample file.